server/src/main/java/org/candlepin/dto/api/v1/DeletedConsumerTranslator.java

```diff
@@ -15,14 +15,16 @@
 package org.candlepin.dto.api.v1;
 
 import org.candlepin.dto.ModelTranslator;
-import org.candlepin.dto.TimestampedEntityTranslator;
+import org.candlepin.dto.ObjectTranslator;
 import org.candlepin.model.DeletedConsumer;
 
+import java.time.ZoneOffset;
+import java.util.Date;
+
 /**
  * This translator provides translation from DeletedConsumer model objects to DeletedConsumerDTOs
  */
-public class DeletedConsumerTranslator extends
-    TimestampedEntityTranslator<DeletedConsumer, DeletedConsumerDTO> {
+public class DeletedConsumerTranslator implements ObjectTranslator<DeletedConsumer, DeletedConsumerDTO> {
```

Comment:
Is this the trend we're following to solve the problem of "DeletedConsumerDTO is not within its bound; should extend `org.candlepin.dto.TimestampedCandlepinDTO`"? Turning the `extends TimestampedEntityTranslator` to `implements ObjectTranslator`?

Reply:
I believe so. I've done a bit of tinkering now, and I can't get the API generation to do any kind of inheritance that's worth considering, which leaves is us with composition; and that, unfortunately, means manually doing all the shared things like timestamping.